packages/web/package.json

```diff
@@ -58,7 +57,7 @@
     "@audius/stems": "*",
     "@audius/trpc-server": "*",
     "@coinbase/cbpay-js": "1.2.0",
-    "@craco/craco": "7.0.0-alpha.3",
+    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
```

Comment:
Do we want to pin this? Also, what this?

Reply:
Yes will pin! We use it to polyfill `buffer`